### ISSUE TITLE
Submit authorisation with enter

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/web#2048]
## Description
On the authorisation page user can't hit enter to start the authorisation. Needs to be possible.

**Comment by marcus-herrmann** [comment]
Cannot reproduce on latest Safari, Firefox, Chrome (all Mac). Was this issue solved in the mean time?

The focus state is very subtle though. Maybe this caused the misunderstanding?

**Comment by LukasHirt** [comment]
This should happen without needing to send there the focus, shouldn't it? Simply load the authorise page and hit enter without hitting anything else. This works for example on the authorise page in old webUI.

**Comment by marcus-herrmann** [comment]
My focus style assumption was the following: maybe you pressed Enter while actually the focus wasn't on the button. Interestingly, there is a hidden tab-stop (focussable item) between Authorize and Switch users.

But yes, what you described works in the above mentioned systems. "Authorize" gets autofocussed by the autofocus attribute. Loading + hitting Enter right away works on my machine.

**Comment by pascalwengerter** [comment]
@LukasHirt is that a `web` or `idp (oCIS)` issue, and is it still relevant after we've merged the `a11y`-enhancements? :)

**Comment by LukasHirt** [comment]
> is that a web or idp (oCIS) issue,

Both, it's the actual Web login screen

![Screenshot 2021-06-15 at 14 06 38](https://user-images.githubusercontent.com/25989331/122049638-f0baa980-cde2-11eb-8802-dac4b9edfe81.png)


> is it still relevant

Depends... it is possible to focus manually the login button and then hit enter. But there is the skip-to-main link before that which doesn't really make sense since there is only main content

**Comment by JanAckermann** [comment]
I don't think we have this extra step anymore with ocis and username password enter should be sent when pressing enter

**Comment by kulmann** [comment]
Consequently we should remove all traces of that "login redirect info page", see https://github.com/owncloud/web/issues/10456
